addons/account/static/src/scss/backend_section_and_note.scss

```diff
@@ -0,0 +1,32 @@
+/*
+    The goal of this file is to contain CSS hacks related to allowing
+    section and note on sale order and invoice.
+*/
+div[name='order_line'],
```

Comment:
This is not very modular, as sale is not installed with account...

Reply:
I'm not sure I understand this comment, because account is a dependency of sale (?)

odoo/addons/sale/__manifest__.py line 12